sources/packages/backend/libs/test-utils/src/factories/confirmation-of-enrollment.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  Application,
+  ApplicationStatus,
+  COEStatus,
+  DisbursementSchedule,
+  DisbursementScheduleStatus,
+  DisbursementValue,
+  DisbursementValueType,
+  EducationProgramOffering,
+  Institution,
+  InstitutionLocation,
+  OfferingIntensity,
+  Student,
+  StudentAssessment,
+  User,
+} from "@sims/sims-db";
+import { addDays, getISODateOnlyString } from "@sims/utilities";
+import { DataSource } from "typeorm";
+import { createFakeApplication } from "./application";
+import { createFakeDisbursementSchedule } from "./disbursement-schedule";
+import { createFakeDisbursementValue } from "./disbursement-value";
+import { createFakeEducationProgramOffering } from "./education-program-offering";
+import { createFakeStudent } from "./student";
+import { createFakeStudentAssessment } from "./student-assessment";
+import { createFakeUser } from "./user";
+
+/**
+ * Create and save to the database a disbursement record with all the dependencies.
+ * @param dataSource manages the repositories to save the data.
+ * @param relations dependencies.
+ * @returns the created application and its dependencies including the disbursement
+ * with the confirmation of enrollment data.
+ */
+export async function saveFakeApplicationCOE(
+  dataSource: DataSource,
+  relations?: {
+    institution?: Institution;
+    institutionLocation?: InstitutionLocation;
+    disbursementValues?: DisbursementValue[];
+  },
+  options?: {
```

Comment:
comment?

Reply:
Yes, I am adding it, same comment from @dheepak-aot . To be honest, if we take a look at the other factory methods, they are mostly missing comments. I am glad to see that everyone is on onboard to add comments to all of them 😉 
@andrepestana-aot @sh16011993 FYI